src/js/components/Menu/stories/MenuCustomIcons.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import { Grommet, Box, Menu } from 'grommet';
+import { grommet } from 'grommet/themes';
+import { Menu as HamburgerMenu, Close } from 'grommet-icons';
+import { deepMerge } from '../../../utils';
+
+const customTheme = deepMerge(grommet, {
+  menu: {
+    icons: {
+      down: HamburgerMenu,
+      up: Close,
```

Comment:
I'm wondering if we should provide a more usable icons-example, I'm not sure the Close icon is semantically accurate for the Menu use case. What type of icons are you using in your app?
If you don't have another alternative to suggest we should maybe consider showing an example like Select with FormUp/FormDown icons.

Reply:
For me, it's a common use case of icons for the menu, and in my app I use it. But it's not the principal for me, what kind of icons will be in the example, the main thing is the functionality. So I changed icons to FormUp/FormDown in the latest commit.